packages/contracts/contracts/voting/majority/MajorityVotingBase.sol

```diff
@@ -13,8 +13,14 @@ import {IDAO} from "../../core/IDAO.sol";
 
 /// @title MajorityVotingBase
 /// @author Aragon Association - 2022
-/// @notice The abstract implementation of majority voting components.
-/// @dev This component implements the `IMajorityVoting` interface.
+/// @notice The abstract implementation of majority voting plugin. We use the following definitions:
+/// - Relative support: `N_yes / (N_yes + N_no)`
+/// - Total support   : `N_yes/ N_total`
```

Comment:
add space to /// - Total support   : `N_yes/ N_total`
like: /// - Total support   : `N_yes / N_total`
same for IMajorityVoting.sol

Reply:
Done